### DISCUSSION TITLE
Consider making the default prayer pace 2-3 minutes?

### DISCUSSION THREAD

**Discussion by ChrisChasm** [Pray4Movement/prayer-global-porch#29] [Ideas]
Right now the default prayer pace is 1 minute. This makes sense for the challenge. You can change your default pace to slow it down if you want in the options.

But is it worth considering making the default pace 2 minutes or 3 minutes?

1. Experientially, we could expand the amount of content before the "more content" fold. 
2. This would give people time to focus and ponder
3. The could still speed up the prayer pace in the options like now.
4. This would generate more quantity of prayer and lower quantity of locations.

**Comment by squigglybob** [comment]
This sounds like a good idea to me. 1 min feels to quick, and it would have the benefits that you mentioned in point 4

**Comment by ChrisChasm** [comment]
The negative implications are that the challenge concept with the Moravians leans on the 1 minute language, the math is simpliest in 1 minute blocks (2-3 minutes is a harder story to tell both on the maps, etc.)
